clang/include/clang/Driver/Options.td

```diff
@@ -969,6 +969,10 @@ def Xlinker : Separate<["-"], "Xlinker">, Flags<[LinkerInput, RenderAsInput]>,
   Visibility<[ClangOption, CLOption, FlangOption]>,
   HelpText<"Pass <arg> to the linker">, MetaVarName<"<arg>">,
   Group<Link_Group>;
+def Xdist : Separate<["-"], "Xdist">, Flags<[LinkOption]>,
```

Comment:
Not sure I am a fan of Xdist here. Since the feature is called DTLTO why not Xdtlo, dist is a bit ambiguous.

Reply:
Good comment, thanks. In this implementation, there are two things to which a user might want to pass arguments:

1. A distributor – which is whatever process is invoked to consume the JSON and distribute the ThinLTO backend compilations.
2. Whatever tool will be invoked by the distributors to perform the ThinLTO backend compilations – in this PR, that is currently `clang`; however, we refer to this as the `remote optimization tool` since it is possible to swap to a different tool without changing the distributors.

We went with `-Xdist` as it indicates that the arguments will be forwarded to the distributor process and is short (to reduce verbosity if there are many forwarded arguments). What would you think about `-Xthinlto-distributor=` instead?

Reply:
I think being explicit is preferred for compiler options.

Reply:
Agreed that `-Xdist` is confusing. -Xthinlto-distributor looks good. If someone can come with up with a shorter, non-confusing one, that'd be nice:)

Reply:
I'll wait another day and then change to `-Xthinlto-distributor` if there isn't a better suggestion :)

Reply:
I have updated to use `-Xthinlto-distributor`. It's longer than `-Xdist` so I have used `CommaJoined`  which allows e.g. `-fthinlto-distributor=incredibuild.exe -Xthinlto-distributor=--verbose,--j10`